app/models/good_job/execution.rb

```diff
@@ -455,12 +459,14 @@ def perform
         job_error = result.handled_error || result.unhandled_error
         if job_error
           error_string = self.class.format_error(job_error)
+          error_backtrace = self.class.format_backtrace(job_error.backtrace)
 
           job_attributes[:error] = error_string
           job_attributes[:error_event] = result.error_event if self.class.error_event_migrated?
           if discrete_execution
             discrete_execution.error = error_string
             discrete_execution.error_event = result.error_event
+            discrete_execution.error_backtrace = error_backtrace if discrete_execution.class.backtrace_migrated?
```

Comment:
Just to save a little bit of unnecessary processing, let's only format the backtrace if it's been migrated:

```suggestion
            if discrete_execution.class.backtrace_migrated?
              error_backtrace = self.class.format_backtrace(job_error.backtrace)
              discrete_execution.error_backtrace = error_backtrace
            end
```